packages/drivers/routerlicious-driver/src/restWrapper.ts

```diff
@@ -159,17 +175,22 @@ export class RouterliciousStorageRestWrapper extends RouterliciousRestWrapper {
                         documentId,
                         refreshToken
                     );
-                    const credentials = {
-                        password: storageToken.jwt,
-                        user: tenantId,
-                    };
-                    return getAuthorizationTokenFromCredentials(credentials);
+
+                    return storageToken;
                 }
             );
         };
 
+        const getAuthorizationHeader: AuthorizationHeaderGetter = (token?: ITokenResponse): string => {
```

Comment:
nit: I don't see a case where token can be undefined when passed into this function. Make it non-optional?